cosmos/ingestion/ingest/ingest.py

```diff
@@ -91,7 +92,8 @@ def ingest(self,
                enrich=False,
                pp_threshold=0.8,
                d_threshold=-10,
-               spans=20):
+               spans=20,
```

Comment:
It looks like we're inconsistent with how we're handling a lot of these parameters. Some pipeline toggles (postprocess, detection) are turned on and off at the class level, while others (word vecs, ocr skipping, enrich) are done when calling `Ingest.ingest`. Is there a particular reason you've done it in `Ingest.ingest`?

Reply:
Potentially use class properties here.

Reply:
@iross moved enrich and qa switches to class properties, and changed up the cli options to be consistent with other behavior modifying switches.